aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -750,18 +836,19 @@ module aptos_framework::account {
     }
 
     /// create the account for system reserved addresses
-    public(friend) fun create_framework_reserved_account(addr: address): (signer, SignerCapability) {
+    public(friend) fun create_framework_reserved_account(addr: address)
+        : (signer, SignerCapability) {
         assert!(
-            addr == @0x1 ||
-                addr == @0x2 ||
-                addr == @0x3 ||
-                addr == @0x4 ||
-                addr == @0x5 ||
-                addr == @0x6 ||
-                addr == @0x7 ||
-                addr == @0x8 ||
-                addr == @0x9 ||
-                addr == @0xa,
+            addr == @0x1
+            || addr == @0x2
+            || addr == @0x3
+            || addr == @0x4
+            || addr == @0x5
+            || addr == @0x6
+            || addr == @0x7
+            || addr == @0x8
+            || addr == @0x9
+            || addr == @0xa,
```

Comment:
/Does this still compile? Sometimes `||` at the beginning of the line doesn't compile correctly in languages.

Reply:
Sure, that's no problem. We have conducted tests, and under the 'aptos-core' directory, the formatting is executed multiple times in a loop.